server/mailet/mailets/src/test/java/org/apache/james/transport/mailets/RecipientRewriteTableTest.java

```diff
@@ -93,14 +97,21 @@ public void testAddressMapping() throws Exception {
      * @throws MessagingException
      */
     private Mail createMail(String[] recipients) throws MessagingException {
-        Mail mail = new FakeMail();
-        ArrayList<MailAddress> a = new ArrayList<MailAddress>(recipients.length);
-        for (String recipient : recipients) {
-            a.add(new MailAddress(recipient));
-        }
-        mail.setRecipients(a);
-        mail.setMessage(new MimeMessage(NO_SESSION));
-        return mail;
+        return FakeMail.builder()
```

Comment:
No, it belongs to this commit because of re-factoring. It should be completely removed at another PR by RRT re-factoring